src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -525,14 +568,20 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(makeQRInfoScreen(supportedCountries: supportedCountries, testRegistrationInformation: testRegistrationInformation))
 	}
 
-	private func showFamilyMemberTestConsentScreen() {
+	private func showFamilyMemberTestConsentScreen(testRegistrationInformation: CoronaTestRegistrationInformation) {
 		let familyMemberConsentViewController = FamilyMemberConsentViewController(
 			dismiss: { [weak self] in
 				self?.dismiss()
 			}, didTapDataPrivacy: { [weak self] in
 				self?.showDataPrivacy()
-			}, didTapSubmit: { givenName in
+			}, didTapSubmit: { [weak self] givenName in
 				Log.info("User has give name \(givenName)")
+				self?.showTestCertificateScreenIfNecessary(
+					testOwner: .familyMember(displayName: givenName),
+					testRegistrationInformation: testRegistrationInformation,
+					submissionConsentGiven: nil,
```

Comment:
Is 'nil' correct here? Don't we have a decision by the user at the point, since its the callback of 'didTapSubmit'?

Reply:
Yes it is correct, we don't have the consent for submission in the case of family member as we are not submitting the keys.